_data/projects/baker.yml

```
@@ -0,0 +1,16 @@
+---
+name: Baker
+desc: Baker is a library that reduces the effort to orchestrate (micro)service-based process flows
+site: https://ing-bank.github.io/baker/
+tags:
+  - java
+  - scala
+  - petri-nets
+  - jvm
+  - micro-service
+upforgrabs:
+  name: up-for-grabs
```

Keeping these labels in sync with what's on GitHub:

```suggestion
  name: Hacktoberfest
```